src/Documentation/grains/index.md

```diff
@@ -65,10 +65,49 @@ public class PlayerGrain : Grain, IPlayerGrain
 }
 ```
 
+Grains can have two interfaces and one implementation.
+
+```csharp
+namespace UserNotification.Host
+{
+    public class MatchMakingGrain : Grain, IGameGrain, IPlayerDisconnect
+    {
+      // ...
+    }
+    var grainClassNamePrefix = "UserNotification.Host.MatchMakingGrain";
+    var sameActivationDifferentFace1 = this.GrainFactory.GetGrain<IGameGrain>("13", grainClassNamePrefix);
+    var sameActivationDifferentFace1  = this.GrainFactory.GetGrain<IPlayerDisconnect>("13", grainClassNamePrefix);
+}
+```
+
+If you have a grain which implements multiple distinct interfaces, like so:
+
+```csharp
+class MegaGrain : Grain, IUserGrain, IMonkeyGrain 
```

Comment:
Where is this from - MegaGrain, IMonkeyGrain? What does this mean?

Reply:
I got that from @ReubenBond's reply on #1511 https://github.com/dotnet/orleans/issues/1511.